test/credo_check/migrations_safety_test.exs

```diff
@@ -16,6 +16,17 @@ defmodule ExcellentMigrations.CredoCheck.MigrationsSafetyTest do
     |> assert_issues()
   end
 
+  test "it should report code that is defined in user specified directory" do
+    Application.put_env(:excellent_migrations, :migrations_paths, [
```

Comment:
Using `Application.put_env` requires restoring the original value after the test is finished ([example](https://elixirforum.com/t/using-application-get-env-application-put-env-in-exunit-tests/8019/3)). Otherwise the new value will stay there and will affect other tests. Maybe the problem isn't manifested in any way now, but in the future that might change. 

Also because it means editing global config, it may no longer be possible to run test asynchronously. I don't recommend the approach with `Application.put_env`. Instead I propose:

1. Passing `migrations_paths` to `FilesFinder.get_migrations_paths` as an argument. This way the test won't need to set the config.
1. Or wrapping interaction with config in a new module with interface and create mock for tests with different implementation.

I'd go with the first approach – I believe it's fine & simple.